custom/icds_reports/sqldata/exports/system_usage.py

```diff
@@ -92,14 +88,43 @@ def columns(self):
                 'Number of take home rations forms',
                 SumColumn('usage_num_thr'),
                 slug='num_thr_forms'
-            ),
-            AggregateColumn(
+            )
+        ]
+        if self.beta:
+            agg_columns.append(DatabaseColumn(
```

Comment:
it looks it changes the order of columns? could we make sure the order is not changed.

Reply:
we can do that. But the addition will effect the readability